cmd/manager/main.go

```diff
@@ -63,7 +67,13 @@ func main() {
 	features.OperatorControllerFeatureGate.AddFlag(pflag.CommandLine)
 	pflag.Parse()
 
+	if operatorControllerVersion {
+		fmt.Println(version.String())
+		os.Exit(0)
+	}
+
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts), zap.StacktraceLevel(zapcore.DPanicLevel)))
+	setupLog.Info("starting up the provisioner", "Git commit", version.String())
```

Comment:
Nit: Its no longer a `provisioner` (as in rukpak), I'd make it a generic `controller`
```suggestion
	setupLog.Info("starting up the controller", "Git commit", version.String())
```